wsmaster/che-core-api-ssh/src/main/java/org/eclipse/che/api/ssh/server/jpa/JpaSshDao.java

```diff
@@ -0,0 +1,120 @@
+        requireNonNull(sshPair);
+        try {
+            doCreate(sshPair);
+        } catch (DuplicateKeyException e) {
+            throw new ConflictException(String.format("Ssh pair with service '%s' and name '%s' already exists",
+                                                      sshPair.getService(),
+                                                      sshPair.getName()));
+        } catch (RuntimeException e) {
+            throw new ServerException(e);
+        }
+    }
+
+    @Override
+    @Transactional
+    public List<SshPairImpl> get(String owner, String service) throws ServerException {
+        requireNonNull(owner);
+        requireNonNull(service);
+        try {
+            return managerProvider.get()
+                                  .createQuery("SELECT pair " +
+                                               "FROM SshKeyPair pair " +
+                                               "WHERE pair.owner = :owner " +
+                                               "  AND pair.service = :service", SshPairImpl.class)
+                                  .setParameter("owner", owner)
+                                  .setParameter("service", service)
+                                  .getResultList();
+
+        } catch (RuntimeException e) {
+            throw new ServerException(e.getLocalizedMessage(), e);
+        }
+    }
+
+    @Override
+    @Transactional
+    public SshPairImpl get(String owner, String service, String name) throws ServerException, NotFoundException {
+        requireNonNull(owner);
+        requireNonNull(service);
+        requireNonNull(name);
+        try {
+            SshPairImpl result = managerProvider.get().find(SshPairImpl.class, new SshPairPrimaryKey(owner, service, name));
+            if (result == null) {
+                throw new NotFoundException(String.format("Ssh pair with service '%s' and name '%s' was not found.", service, name));
+            }
+            return result;
+
+        } catch (RuntimeException e) {
+            throw new ServerException(e.getLocalizedMessage(), e);
+        }
+    }
+
+    @Override
+    public void remove(String owner, String service, String name) throws ServerException, NotFoundException {
+        requireNonNull(owner);
+        requireNonNull(service);
+        requireNonNull(name);
+        try {
+            doRemove(owner, service, name);
+        } catch (RuntimeException e) {
+            throw new ServerException(e);
+        }
+    }
+
+    @Transactional
+    protected void doCreate(SshPairImpl entity) {
+        managerProvider.get().persist(entity);
+    }
+
+    @Transactional
+    protected void doRemove(String owner, String service, String name) throws NotFoundException {
+        EntityManager manager = managerProvider.get();
+        SshPairImpl entity = manager.find(SshPairImpl.class, new SshPairPrimaryKey(owner, service, name));
+        if (entity != null) {
+            manager.remove(entity);
+        } else {
```

Comment:
i'd change the code to avoid else block, just throw exception when the entity is null, it looks better for me